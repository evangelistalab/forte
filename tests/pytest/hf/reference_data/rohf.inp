molecule {
0 3
C
H 1 1.085
H 1 1.085 2 135.5
}

set scf_type pk
set basis cc-pVDZ
set reference rohf

energy('scf')
