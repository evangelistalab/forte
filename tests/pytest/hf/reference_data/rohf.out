
    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                               Psi4 undefined 

                         Git: Rev {master} ff26262 


    D. G. A. Smith, L. A. Burns, A. C. Simmonett, R. M. Parrish,
    M. C. Schieber, R. Galvelis, P. Kraus, H. Kruse, R. Di Remigio,
    A. Alenaizan, A. M. James, S. Lehtola, J. P. Misiewicz, M. Scheurer,
    R. A. Shaw, J. B. Schriber, Y. Xie, Z. L. Glick, D. A. Sirianni,
    J. S. O'Brien, J. M. Waldrop, A. Kumar, E. G. Hohenstein,
    B. P. Pritchard, B. R. Brooks, H. F. Schaefer III, A. Yu. Sokolov,
    K. Patkowski, A. E. DePrince III, U. Bozkaya, R. A. King,
    F. A. Evangelista, J. M. Turney, T. D. Crawford, C. D. Sherrill,
    J. Chem. Phys. 152(18) 184108 (2020). https://doi.org/10.1063/5.0006002

                            Additional Code Authors
    E. T. Seidl, C. L. Janssen, E. F. Valeev, M. L. Leininger,
    J. F. Gonthier, R. M. Richard, H. R. McAlexander, M. Saitow, X. Wang,
    P. Verma, and M. H. Lechner

             Previous Authors, Complete List of Code Contributors,
                       and Citations for Specific Modules
    https://github.com/psi4/psi4/blob/master/codemeta.json
    https://github.com/psi4/psi4/graphs/contributors
    http://psicode.org/psi4manual/master/introduction.html#citing-psifour

    -----------------------------------------------------------------------


    Psi4 started on: Thursday, 17 June 2021 01:35PM

    Process ID: 70464
    Host:       CHE-FEVANGE
    PSIDATADIR: /Users/fevange/Source/psi4/objdir-Debug/stage/share/psi4
    Memory:     500.0 MiB
    Threads:    1
    
  ==> Input File <==

--------------------------------------------------------------------------
molecule {
0 3
C
H 1 1.085
H 1 1.085 2 135.5
}

set scf_type pk
set basis cc-pVDZ
set reference rohf

energy('scf')
--------------------------------------------------------------------------

Scratch directory: /tmp/

*** tstart() called on CHE-FEVANGE
*** at Thu Jun 17 13:35:04 2021

   => Loading Basis Set <=

    Name: CC-PVDZ
    Role: ORBITAL
    Keyword: BASIS
    atoms 1   entry C          line   138 file /Users/fevange/Source/psi4/objdir-Debug/stage/share/psi4/basis/cc-pvdz.gbs 
    atoms 2-3 entry H          line    22 file /Users/fevange/Source/psi4/objdir-Debug/stage/share/psi4/basis/cc-pvdz.gbs 


         ---------------------------------------------------------
                                   SCF
               by Justin Turney, Rob Parrish, Andy Simmonett
                          and Daniel G. A. Smith
                             ROHF Reference
                        1 Threads,    500 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c2v
    Full point group: C2v

    Geometry (in Angstrom), charge = 0, multiplicity = 3:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
         C            0.000000000000     0.000000000000    -0.059083743566    12.000000000000
         H            0.000000000000    -1.004211446859     0.351750006261     1.007825032230
         H            0.000000000000     1.004211446859     0.351750006261     1.007825032230

  Running in c2v symmetry.

  Rotational constants: A =     57.87368  B =      8.29337  C =      7.25388 [cm^-1]
  Rotational constants: A = 1735009.41259  B = 248628.96811  C = 217465.84665 [MHz]
  Nuclear repulsion =    6.116130155539318

  Charge       = 0
  Multiplicity = 3
  Electrons    = 8
  Nalpha       = 5
  Nbeta        = 3

  ==> Algorithm <==

  SCF Algorithm Type is PK.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is SAD.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 1.00e-12

  ==> Primary Basis <==

  Basis Set: CC-PVDZ
    Blend: CC-PVDZ
    Number of shells: 12
    Number of basis functions: 24
    Number of Cartesian functions: 25
    Spherical Harmonics?: true
    Max angular momentum: 2

  ==> Integral Setup <==

  Using in-core PK algorithm.
   Calculation information:
      Number of atoms:                   3
      Number of AO shells:              12
      Number of primitives:             32
      Number of atomic orbitals:        25
      Number of basis functions:        24

      Integral cutoff                 1.00e-12
      Number of threads:                 1

  Performing in-core PK
  Using 90300 doubles for integral storage.
  We computed 3081 shell quartets total.
  Whereas there are 3081 unique shell quartets.

  ==> DiskJK: Disk-Based J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    Memory [MiB]:              375
    Schwarz Cutoff:          1E-12

    OpenMP threads:              1

  Minimum eigenvalue in the overlap matrix is 2.9663535828E-02.
  Reciprocal condition number of the overlap matrix is 9.2981101216E-03.
    Using symmetric orthogonalization.

  ==> Pre-Iterations <==

  SCF Guess: Superposition of Atomic Densities via on-the-fly atomic UHF (no occupation information).

   -------------------------
    Irrep   Nso     Nmo    
   -------------------------
     A1        11      11 
     A2         2       2 
     B1         4       4 
     B2         7       7 
   -------------------------
    Total      24      24
   -------------------------

  ==> Iterations <==

                        Total Energy        Delta E     RMS |[F,P]|

   @ROHF iter SAD:   -38.16933119934495   -3.81693e+01   0.00000e+00 
   @ROHF iter   1:   -38.91100789966510   -7.41677e-01   5.75387e-03 DIIS
   @ROHF iter   2:   -38.91996467117516   -8.95677e-03   1.85417e-03 DIIS
   @ROHF iter   3:   -38.92101374862091   -1.04908e-03   4.15852e-04 DIIS
   @ROHF iter   4:   -38.92106985433191   -5.61057e-05   1.27537e-04 DIIS
   @ROHF iter   5:   -38.92107468928804   -4.83496e-06   1.92409e-05 DIIS
   @ROHF iter   6:   -38.92107481889126   -1.29603e-07   3.40646e-06 DIIS
   @ROHF iter   7:   -38.92107482288969   -3.99842e-09   5.49900e-07 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

    Orbital Energies [Eh]
    ---------------------

    Doubly Occupied:                                                      

       1A1   -11.247312     2A1    -0.847079     1B2    -0.598726  

    Singly Occupied:                                                      

       3A1    -0.142802     1B1    -0.103469  

    Virtual:                                                              

       4A1     0.208147     2B2     0.264381     5A1     0.608683  
       3B2     0.702378     2B1     0.719787     6A1     0.731545  
       4B2     0.919279     7A1     1.016661     1A2     1.191117  
       8A1     1.356097     3B1     1.371726     5B2     1.539469  
       9A1     1.751291     4B1     1.876944     6B2     2.122750  
       2A2     2.210864    10A1     2.223457     7B2     2.757357  
      11A1     2.948358  

    Final Occupation by Irrep:
             A1    A2    B1    B2 
    DOCC [     2,    0,    0,    1 ]
    SOCC [     1,    0,    1,    0 ]

  @ROHF Final Energy:   -38.92107482288969

   => Energetics <=

    Nuclear Repulsion Energy =              6.1161301555393184
    One-Electron Energy =                 -63.8272906378609122
    Two-Electron Energy =                  18.7900856594319130
    Total Energy =                        -38.9210748228896790

Computation Completed


Properties will be evaluated at   0.000000,   0.000000,   0.000000 [a0]

Properties computed using the SCF density matrix

  Nuclear Dipole Moment: [e a0]
     X:     0.0000      Y:     0.0000      Z:     0.6595

  Electronic Dipole Moment: [e a0]
     X:     0.0000      Y:     0.0000      Z:    -0.4497

  Dipole Moment: [e a0]
     X:     0.0000      Y:     0.0000      Z:     0.2098     Total:     0.2098

  Dipole Moment: [D]
     X:     0.0000      Y:     0.0000      Z:     0.5334     Total:     0.5334


*** tstop() called on CHE-FEVANGE at Thu Jun 17 13:35:05 2021
Module time:
	user time   =       0.61 seconds =       0.01 minutes
	system time =       0.05 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes
Total time:
	user time   =       0.61 seconds =       0.01 minutes
	system time =       0.05 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes

    Psi4 stopped on: Thursday, 17 June 2021 01:35PM
    Psi4 wall time for execution: 0:00:00.93

*** Psi4 exiting successfully. Buy a developer a beer!
